src/v2/Components/NavBar/Menus/UserMenu.tsx

```diff
@@ -39,39 +42,59 @@ export const UserMenu: React.FC = () => {
   const hasPartnerAccess = Boolean(user.has_partner_access)
 
   return (
-    <Menu onClick={trackClick}>
-      {isAdmin && <MenuItem href={sd.ADMIN_URL}>Admin</MenuItem>}
-      {(isAdmin || hasPartnerAccess) && (
-        <MenuItem href={sd.CMS_URL}>CMS</MenuItem>
+    <Menu m={0}>
+      {isAdmin && (
+        <MenuItem href={sd.ADMIN_URL} onClick={trackClick}>
+          Admin
+        </MenuItem>
       )}
+
       {(isAdmin || hasPartnerAccess) && (
-        <Flex width="100%" justifyContent="center" my={1}>
-          <Box width="90%">
-            <Separator />
-          </Box>
-        </Flex>
+        <>
+          <MenuItem href={sd.CMS_URL} onClick={trackClick}>
+            CMS
+          </MenuItem>
+
+          <Flex width="100%" justifyContent="center" my={1}>
+            <Box width="90%">
+              <Separator />
+            </Box>
+          </Flex>
+        </>
       )}
+
       {isAdmin && (
-        <MenuItem href="/user/purchases">
-          <TagIcon mr={1} /> Purchases
+        <MenuItem href="/user/purchases" onClick={trackClick}>
+          <TagIcon mr={1} title="View your purchases" /> Purchases
         </MenuItem>
       )}
-      <MenuItem href="/user/saves">
-        <HeartIcon mr={1} /> Saves & Follows
+
+      <MenuItem href="/user/saves" onClick={trackClick}>
+        <HeartIcon mr={1} title="View your Saves &amp; Follows" /> Saves &amp;
+        Follows
       </MenuItem>
-      <MenuItem href="/profile/edit">
-        <SoloIcon mr={1} /> Collector Profile
+
+      <MenuItem href="/profile/edit" onClick={trackClick}>
+        <SoloIcon mr={1} title="View your Collector Profile" /> Collector
+        Profile
       </MenuItem>
-      <MenuItem href="/user/edit">
-        <SettingsIcon mr={1} /> Settings
+
+      <MenuItem href="/user/edit" onClick={trackClick}>
+        <SettingsIcon mr={1} title="Edit your settings" /> Settings
       </MenuItem>
+
       <MenuItem
-        onClick={event => {
-          event.preventDefault() // `href` is only for tracking purposes
+        tabIndex={0}
+        onKeyPress={event => {
+          if (event.key === "Enter" || event.key === " ") {
+            mediator.trigger("auth:logout")
+          }
+        }}
```

Comment:
So this is how you handle accessibility for non buttons. Really it's simpler to use a button and the `Clickable` component inside of palette will give you one that essentially acts like a div with all the styles stripped. But if you want to use an anchor tag or a div, you need to give it a tabIndex of 0 so that it's focusable, then handle the keyPress.